step-plans/step-plans-base-artefacts/src/main/java/step/reporting/JUnit4ReportWriter.java

```diff
@@ -142,20 +143,26 @@ private ReportGenerationResult writeSingleTestSuiteXml(ReportTreeAccessor report
 				"failures=\"" + numberOfFailures.get() + "\" " +
 				"errors=\"" + numberOfErrors.get() + "\">");
 		writer.write('\n');
-		writer.write("<properties>");
-		if (junit4ReportConfig.isAddLinksToStepFrontend()) {
-			String linkToStep = generateLinkToStep(executionId);
-			if (linkToStep != null) {
-				writer.write('\n');
-				writer.write("<property name=\"url:step\" value=\"" + linkToStep + "\"/>");
-				writer.write('\n');
-			}
-		}
-		writer.write("</properties>");
-		writer.write('\n');
-
-		AtomicBoolean errorWritten = new AtomicBoolean(false);
 
+//		Some systems support custom properties for testsuite, but Gitlab doesn't use and display them
```

Comment:
@iegorov777 I would still keep the properties even if gitlab do not display them, they might work with other integration so it doesn't hurt. Otherwise commented code should be removed.

Reply:
@david-stephan 
Agree. I've uncommented this section